src/main/java/com/kapeta/spring/config/KapetaSecurityProviderConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 Kapeta Inc.
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.kapeta.spring.config;
+
+import com.kapeta.spring.rest.KapetaAuthenticationRestController;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
+@EnableMethodSecurity
+public class KapetaSecurityProviderConfig {
```

Comment:
Aren't we missing some things wrt JWKS ? https://www.baeldung.com/spring-security-oauth2-jws-jwk